tools/bundle_adjust_tracks.cxx

```diff
@@ -1018,6 +1021,26 @@ static int maptk_main(int argc, char const* argv[])
   //
   // Write the output KRTD files
   //
+  if (config->get_value<bool>("krtd_clean_up") )
+  {
+
+    std::cerr << "Cleaning "
+              << config->get_value<std::string>("output_krtd_dir")
+              << " before writing new files." << std::endl;
+
+    kwiver::vital::path_t krtd_dir = config->get_value<std::string>("output_krtd_dir");
+    std::vector<kwiver::vital::path_t> files = files_in_dir(krtd_dir);
+
+    for (int i = 0; i < files.size(); ++i)
+    {
+      if (ST::GetFilenameExtension(files[i]) == ".krtd")
```

Comment:
There is a subtle bug here that we've been bitten by before.  `ST::GetFilenameExtension()` returns the **longest** filename extension.  So if your file is named `image.0001.krtd` the extension returned is `.0001.krtd`.  Instead if you use `ST::GetFilenameLastExtension()` it would return `.krtd` for this same example.

Otherwise, this branch looks good.